ports/glutin/window.rs

```diff
@@ -1397,6 +1403,44 @@ fn filter_nonprintable(ch: char, key_code: VirtualKeyCode) -> Option<char> {
     }
 }
 
+fn ctrl_key_to_key(ch: char) -> char {
+    match ch {
+        '\u{0}' => '@',
+        '\u{1}' => 'a',
+        '\u{2}' => 'b',
+        '\u{3}' => 'c',
+        '\u{4}' => 'd',
+        '\u{5}' => 'e',
+        '\u{6}' => 'f',
+        '\u{7}' => 'g',
+        '\u{8}' => 'h',
+        '\t' => 'i',
+        '\n' => 'j',
+        '\u{b}' => 'k',
+        '\u{c}' => 'l',
+        '\r' => 'm',
+        '\u{e}' => 'n',
+        '\u{f}' => 'o',
+        '\u{10}' => 'p',
+        '\u{11}' => 'q',
+        '\u{12}' => 'r',
+        '\u{13}' => 's',
+        '\u{14}' => 't',
+        '\u{15}' => 'u',
+        '\u{16}' => 'v',
+        '\u{17}' => 'w',
+        '\u{18}' => 'x',
+        '\u{19}' => 'y',
+        '\u{1a}' => 'z',
+        '\u{1b}' => '[',
+        '\u{1c}' => '\\', //Need to escape \
+        '\u{1d}' => ']',
+        '\u{1e}' => '^',
+        '\u{1f}' => '_',
+        _ => ch
+    }
+}
```

Comment:
Do you need more? Like for "Ctrl +" , "Ctrl digit", "Ctrl ," … ? Do we even need to cover everything?